changelogs/fragments/support_cli_syntax_change.yaml

```diff
@@ -0,0 +1,3 @@
+---
+breaking_changes:
+  - Add support for syntax changes in release train 4.23 and after.
```

Comment:
Is 4.23 the new minimum version? I would probably make that a little more explicit.